source/docs/software/vscode-overview/wpilib-commands-vscode.rst

```diff
@@ -5,25 +5,29 @@ This document contains a complete list of the commands provided by the WPILib VS
 To access these commands, press Ctrl+Shift+P to open the Command Palette, then begin typing the command name as shown here to filter the list of commands. Click on the command name to execute it.
 
 - **WPILib: Build Robot Code** - Builds open project using GradleRIO
-- **WPILib: Create Project** - Create a new robot project
+- **WPILib: Create a new project** - Create a new robot project
 - **WPILib C++: Refresh C++ Intellisense** - Force an update to the C++ Intellisense configuration.
 - **WPILib C++: Select current C++ toolchain** - Select the toolchain to use for Intellisense (i.e. desktop vs. roboRIO vs...). This is the same as clicking the current mode in the bottom right status bar.
+- **WPILib C++ Select Enabled C++ Intellisense Binary Types** - Switch Intellisense between static, shared, and executable
```

Comment:
I don't think we should add this if it's going to get removed, plus it doesn't have much use to the user.